### PR TITLE
Updates for SDK

### PR DESCRIPTION
**Pull request by VickMellon**
A few enhances that required for full SDK (https://github.com/zksync-sdk/zksync-go)
Also some handy updates of example for this lib

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ZKSyncSDK for Go usage example
+
+This simple test project show example of usage of imported package.
+
+## Quick run
+
+For macOS and Linux just run make:
```
Comment (StanislavBreadless) [inline example/README.md:7]: Will it work for Windows?

**Review by StanislavBreadless — COMMENTED** [review]
nice

File: example/README.md (hunk 1 above)
Comment (VickMellon) [reply example/README.md:7]: This `makefile` was designed for Mac and Linux only, to run example fast and easy.
Windows users must manually download appropriate zksync-crypto-c library for Windows and link it during compilation/execution (probably, it also will require some changes in example source code, I am not sure about Win particularities)

Comment (StanislavBreadless) [reply example/README.md:7]: It's ok for first release, but we should add tutorial for WIndows users as well in the future:
https://linear.app/matterlabs/issue/ZSDK-58/windows-tutorial-for-go-sdk

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**